### PR TITLE
Tadams/update gateway

### PR DESCRIPTION
**Pull request by StarChaser44**
Updates clark gateway to include new route 
Screen Shot 2021-12-08 at 12 56 44 PM

### CODE REVIEW

**Review by JosueMolinaMorales — APPROVED** [review]

File: src/modules/learning-object-service/relevancy/RelevancyController.ts [hunk 1]
```diff
@@ -321,6 +321,38 @@ export class RelevancyController implements Controller {
...
+     *        schema:
+     *          type: string
+     *        required: true
+     *        description: The evaluator's username
+     *      - in: path
+     *        name: status
+     *        schema:
+     *          type: string
+     *        required: true
+     *        description: The requested status of evaluated learning objects
```
Comment (EVogel1999) [inline src/modules/learning-object-service/relevancy/RelevancyController.ts:343]: Where is the optional query parameter?

**Review by pzalep1 — COMMENTED** [review]
Once Emily's comment is addressed this will be good to go!

**Review by Cwagne17 — COMMENTED** [review]
Just add the optional param then it'll be good.

**Review by EVogel1999 — APPROVED** [review]
Thank you!

**Review by pzalep1 — APPROVED** [review]

**Review by mfranz1 — APPROVED** [review]

**Pull request merged**